File: VB/App_Code/SampleDataSource.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

''' <summary>
''' Summary description for SampleDataSource
''' </summary>
Public Class SampleDataSource
	Public Shared Function GetDataSource() As List(Of SampleDataRow)
		Dim list As New List(Of SampleDataRow)()
		list.Add(New SampleDataRow() With {.Id = 1, .Field1 = False, .Field2 = False, .Field3 = False, .Field4 = False})
		list.Add(New SampleDataRow() With {.Id = 2, .Field1 = False, .Field2 = False, .Field3 = False, .Field4 = False})
		list.Add(New SampleDataRow() With {.Id = 3, .Field1 = False, .Field2 = False, .Field3 = False, .Field4 = False})
		list.Add(New SampleDataRow() With {.Id = 4, .Field1 = False, .Field2 = False, .Field3 = False, .Field4 = False})
		Return list
	End Function
End Class

File: CS/App_Code/SampleDataSource.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

/// <summary>
/// Summary description for SampleDataSource
/// </summary>
public class SampleDataSource
{
    public static List<SampleDataRow> GetDataSource()
    {
        List<SampleDataRow> list = new List<SampleDataRow>();
        list.Add(new SampleDataRow() { Id = 1, Field1 = false, Field2 = false, Field3 = false, Field4 = false });
        list.Add(new SampleDataRow() { Id = 2, Field1 = false, Field2 = false, Field3 = false, Field4 = false });
        list.Add(new SampleDataRow() { Id = 3, Field1 = false, Field2 = false, Field3 = false, Field4 = false });
        list.Add(new SampleDataRow() { Id = 4, Field1 = false, Field2 = false, Field3 = false, Field4 = false });
        return list;
    }
}